components/molecules/imageLocation/components/imageLocation.js

```diff
@@ -4,30 +4,30 @@
 
 // React
 import React, { useEffect, useState } from 'react'
-import { bool, func, string, object } from 'prop-types'
 
 // Style
 import styled from 'styled-components'
 
 // UI
 import { ImageWrapper } from '../../'
+import { ImageLocationProps } from './props'
 
 export const ImageLocation = ({
-  initialCoordinates,
   coordinatesChange,
-  markerStyles,
+  initialCoordinates,
   item,
+  markerStyles,
   show
 }) => {
   const [coordinates, setCoordinates] = useState(initialCoordinates)
 
   useEffect(() => {
-    coordinatesChange && item && coordinatesChange({ coordinates, item })
+    coordinatesChange && item && coordinates && coordinatesChange({ coordinates, item })
   }, [coordinates])
 
   return (
     <StyledImageLocation show={show}>
-      {item && (
+      {item && item.filename && (
```

Comment:
@mhsnprvr2 we use the ? pattern here item?.filename? I think it will work in this space and others in this file - please check